.github/dependabot.yaml

```diff
@@ -4,5 +4,9 @@ updates:
 - package-ecosystem: "github-actions"
   directory: "/"
   schedule:
-    # Check for updates to GitHub Actions every week
-    interval: "weekly"
+    # Check for updates to GitHub Actions every month
+    interval: "monthly"
+  groups:
+    github-actions:
+    patterns:
+    - "*"
```

Comment:
nit: Although both are accepted by yaml probably [add space for array items is better.](https://stackoverflow.com/questions/17014460/yaml-indentation-for-array-in-hash)